### ISSUE TITLE
Depending the Shop, can't really use the Import Orders feature

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#247]
There are multiple problems here, but all depending the qty of orders you're managing.
Here for 7 days it's about 150, and 60 days (the default on BrickStore) it's > 1000.

![image](https://user-images.githubusercontent.com/32078594/108885806-864de580-7608-11eb-838c-fa961ae7a439.png)

I have no idea why it shows a nice list of orders - like for 60 days - but still you have to download 1000 of something (order detail page?).

This only here takes a couple of minutes on DSL, it's lonnnnng.

It also (probably) makes 1000 requests on the BL server, linked to our membership.
I did this on 2 machines to get 2 orders details, so it made 2000 requests, which is already almost the half of BrickLink tolerated requests per day.

2000 requests for 2 orders is a bit excessive ;-)
Even if it caches them - which doesn't seem to - as I just switched back to 60 and have 1145 orders to download again :-(

Much more, you can't cancel this download.
You may close the window, but it'll restart whenever you reopen it.

If you set it to a lower number, say 7 days, then:
* it doesn't care of this new number until it finished the previous download (60 days, here...) OR you close BrickStore
* but then it doesn't permit to download an order dated 8 days ago (while I still have valid ongoing orders dated october 2020).

**I would find convenient that you review the whole process, as (maybe, and if possible):**
* build the list and don't download anything else,
* download if needed for THIS selected order at the very last moment.

Million thanks!

**Comment by rgriebl** [comment]
You just have too many order! Scale down a bit :)
The problem you are seeing is that the XML order download doesn't contain the address. I need this for our shop though, because I generate the invoice via a print template, so I need the buyers address.
What I could do, is to only retrieve the addresses once you actually import the order.

**Comment by paramecie** [comment]
> You just have too many order! Scale down a bit :)

Because I just want to rule the World, of course!
Na, more orders every day but of a smaller value :-(

Anyway, I really don't give a sh^t to see the buyer's address, why would I?

I'd like to download order 14468000 from 4 months ago, or the one I placed 1 month ago.

Muito danke!

**Comment by paramecie** [comment]
BTW - I signaled this Bug to BrickLink in MARCH 2020 (almost 1 year ago) and it's still not fixed :-(
Ticket : 01A-26021263-0001

In short, if you're a VAT seller, sometimes the XML file show a Total amount which is different of 1 cent compared to both Order page and Order e-mail (which are now exact since I debugged this also...)

A question of number of decimals or rounding formula I guess.
Which shouldn't happen if they didn't recalculate every time the amounts, and differently, but apparently they do!

**In short, if you see in your Order List in BrickStore a difference of 1 cent in the total of an order and you're a VAT seller,** it's not BrickStore, it's BrickLink XML which has fuc^ed up calculations...

**Comment by ZZJHONS** [comment]
I noticed that bug with the excel download too...

**Comment by paramecie** [comment]
Ah. And of course you din't open a ticket? How can that be fixed ;-) No problemo amigo.

**Comment by ZZJHONS** [comment]
I do have tickets open about some bugs, most don't have any reply, and they are almost a year open.

I always open them through that website because at least I know that a ticket was created...

**Comment by paramecie** [comment]
Yep, same - 5 tickets opened for a year. I received last waak a direct reply by e-mail from Russell (himself) that one should be fixed very soon! But the ticket isn't even replied...

**Comment by ZZJHONS** [comment]
The other day the order download from previous 2017 was fixed =)

**Comment by paramecie** [comment]
Great, fixed, far easier and faster now!
